csharp/extractor/Semmle.Extraction.Tests/StubGenerator.cs

```diff
@@ -28,15 +29,16 @@ public class MyTest {
         var stub = GenerateStub(source);
 
         // Verify
-        const string expected = @"public class MyTest {
-public static readonly int MyField1;
-public const string MyField2 = default;
-}
-";
+        var expected = string.Join(Environment.NewLine,
```

Comment:
Why do we need to use `string.Join(Environment.NewLine`?

Reply:
It is because the stub visitor uses `TextWriter.WriteLine` which is OS dependant.
We can set the `NewLine` property to always use `\n` then maybe we can get it to work without explicitly joining with `Environment.NewLine`.
Should we try with a solution like this - and also i the production code to _always_ get the same newlines now matter which OS the stubs are generated on?

Reply:
I don't know if we want to change anything. It's just a test, and readability is not terrible either. Another approach would be keeping the test input in a single string to help readability, splitting it by line, and then joining it with `Environment.NewLine`.

But if you'd choose to go ahead with changing the newline, I think that would be the nicest solution. We do have `text=auto eol=lf` in the `.gitattributes`, I think if we used `\n` in the generator then it would generate content that matches what we push.